torch/csrc/distributed/autograd/context/dist_autograd_container.h

```diff
@@ -73,6 +79,15 @@ class TORCH_API DistAutogradContainer {
 
   static DistAutogradContainer& getInstanceInternal();
 
+  // Sends an RPC to the workers that have a context corresponding to passed in
+  // context_id. This function should be called with the lock.
+  void sendReleaseContextRpc(int64_t context_id);
```

Comment:
These methods are added to reduce code duplication between `releaseContext` and `releaseContextIfPresent`.